modules/notification/js/notification_service.js

```diff
@@ -61,7 +62,7 @@
                     notificationList.splice(notifIndex, 1);
                 }
 
-                _callback();
+                _callback(actionClicked);
```

Comment:
I think we need to reset actionClicked to false once it's been used to call the _callback function otherwise it will stay at true forever for example?